File: Player.vb
Public Class Player
    Public Property Name As String
    Public Property Species As String
    Public Property Account As String
    Public Property UUID As Byte()
    Public Property ChatLog As New List(Of String)
    Sub New(ByVal Connect As ClientConnectPacket)
        Name = Connect.Name
        Species = Connect.Species
        Account = Connect.Account
        UUID = Connect.UUID
    End Sub
End Class

File: Player.cs
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Diagnostics;
public class Player
{
	public string Name { get; set; }
	public string Species { get; set; }
	public string Account { get; set; }
	public byte[] UUID { get; set; }
	public List<string> ChatLog { get; set; }
    public ClientConnectPacket Raw { get; set; }
	public Player(ClientConnectPacket Connect)
	{
		Name = Connect.Name;
		Species = Connect.Species;
		Account = Connect.Account;
		UUID = Connect.UUID;
        Raw = Connect;
	}
}
